src/PrestaShopBundle/Form/Admin/Type/MoneyWithSuffixType.php

```diff
@@ -71,10 +71,9 @@ public function configureOptions(OptionsResolver $resolver)
      * Can be used with array_walk().
      *
      * @param string $value
-     * @param int $key not used, it's only here to make this method compatible with array_walk
      * @param string $suffix
      */
-    private function applySuffix(&$value, $key, $suffix)
+    private function applySuffix(&$value, $suffix)
```

Comment:
@FabienPapet I wonder if you have seen a comment inside a doc that this param is to work with array_walk?

Reply:
I Reverted that change

Reply:
thanks, I reviewed the rest and it looks good to me ❤️ very satisfying changes 😎